docs_user/modules/proc_stopping-infrastructure-management-and-compute-services.adoc

```diff
@@ -2,23 +2,21 @@
 
 = Stopping infrastructure management and Compute services
 
-The source cloud's control plane can be decomissioned,
-which is taking down only cloud controllers, database and messaging nodes.
-Nodes that must remain functional are those running the `Compute`, `Storage`,
-or `Networker` roles (in terms of composable roles covered by
-{OpenStackPreviousInstaller} Heat Templates).
+You must stop cloud Controller nodes, database nodes, and messaging nodes on the {rhos_prev_long} {rhos_prev_ver} control plane. However, nodes that are running the Compute, Storage, or Networker roles must remain functional on the control plane.
+
+The following procedure applies to a single node standalone {OpenStackPreviousInstaller} deployment. You must remove conflicting repositories and packages from your Compute hosts. That is required to install libvirt packages when these hosts are adopted as data plane nodes, where modular libvirt daemons are no longer running in podman containers.
 
 .Prerequisites
 
-* Define the following shell variables. The values that are used are examples and refer to a single node standalone {OpenStackPreviousInstaller} deployment. Replace these example values with values that are correct for your environment:
+* Define the following shell variables. Replace the following example values with values that are correct for your environment:
```

Comment:
Maybe:
"Define the shell variables. Modify the following example to suit your environment:"